src/lib/storage/chunk.hpp

```diff
@@ -40,8 +40,10 @@ class Chunk : private Noncopyable {
   // Returns the segment at a given position
   std::shared_ptr<BaseSegment> get_segment(ColumnID column_id) const;
 
+  void print(int col_size, std::ostream& out = std::cout) const;
+
  protected:
-  // Implementation goes here
+  std::vector<std::shared_ptr<BaseSegment>> segments;
```

Comment:
```suggestion
  std::vector<std::shared_ptr<BaseSegment>> _segments;
```
Identifiers of protected members should begin with an underscore :wink: